Sites/GD2/Project 1/Styles.css

```diff
@@ -0,0 +1,105 @@
+img {
+    max-width: 100%
+}
+
+h1 {
+    color: #260648;
+    font-weight: bold;
+    font-size: 40px
+}
+
+bodycolor {
+    color: #F6F6EC
+}
+
+h2 {
+    color: #6B4B8C;
+    font-weight: bold;
+    font-size: 20px;
+    font-alignment: left
```

Comment:
`font-alignment` doesn't exist, what you want is `text-align` - however, the `h2` is set to align left by default, so you can also just delete this line.
```suggestion
```